osgtest/tests/test_060_fetch_crl.py

```diff
@@ -55,7 +55,7 @@ def test_01_fetch_crl(self):
         else:
             self.assertEquals(status, 0, fail)
         count = len(glob.glob(os.path.join('/etc/grid-security/certificates', '*.r[0-9]')))
-        self.assert_(count > 3, True)
+        self.assert_(count > 3, "Expected > 3 crls but found %s" % count)
```

Comment:
```suggestion
        self.assert_(count > 3, f"Expected > 3 crls but found {count}")
```

f-strings are so much nicer to read :)

Reply:
@brianhlin The documentation for osg-test says that the tests are run in python 2.6, which I don't believe supports f-strings. Is that not accurate? https://github.com/opensciencegrid/osg-test#writing-tests

Reply:
Good catch! We made the Python 2 -> 3 conversion a while back https://github.com/opensciencegrid/osg-test/pull/230 so that's just some stale documentation. Feel free to fix that doc while you're at it!